application/lib/view/screens/TasksScreen.dart

```diff
@@ -0,0 +1,275 @@
+    return this.app.tasks;
+  }
+
+  Widget contentLandscape() {
+    List<Widget> notStartedWidgets = convertTasksToWidgets(getTasks(Status.notStarted));
+    List<Widget> workInProgressWidgets = convertTasksToWidgets(getTasks(Status.workInProgress));
+    List<Widget> doneWidgets = convertTasksToWidgets(getTasks(Status.done));
+    return Column(
+      children: <Widget>[
+        _columnHeader(),
+        Row(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: <Widget>[
+            Expanded(
+                flex: 33,
+                child: Container(
+                  alignment: Alignment.topCenter,
+                  child: SingleChildScrollView(
+                    child: Column(
+                      children: notStartedWidgets,
+                    ),
+                  ),
+                  color: Color.fromRGBO(200, 200, 200, 1),
+                )),
+            Expanded(
+                flex: 33,
+                child: Container(
+                  alignment: Alignment.topCenter,
+                  child: SingleChildScrollView(
+                    child: Column(
+                      children: workInProgressWidgets,
+                    ),
+                  ),
+                  color: Colors.white,
+                )),
+            Expanded(
+                flex: 33,
+                child: Container(
+                  alignment: Alignment.topCenter,
+                  child: SingleChildScrollView(
+                    child: Column(
+                      children: doneWidgets,
+                    ),
+                  ),
+                  color: Colors.lightGreen,
+                )),
+          ],
+        )
+      ],
+    );
+  }
+
+  Widget generateAppBarLandscape() {
+    return AppBar(
+      centerTitle: true,
+      title: Text(this.app.appName + " - Select Task"),
+      leading: IconButton(
+        icon: FaIcon(FontAwesomeIcons.arrowLeft),
+        onPressed: () => Routes.pop(this.contextObject.getOutput()),
+      ),
+      actions: <Widget>[
+        createNewTaskButton(),
+      ],
+    );
+  }
+
+  Widget _columnHeader() {
+    return Row(children: <Widget>[
+      Expanded(
+          flex: 33,
+          child: Container(
+            decoration: BoxDecoration(border: Border.all()),
+            alignment: Alignment.topCenter,
+            child: Column(
+              children: <Widget>[
+                FaIcon(FontAwesomeIcons.hourglass),
+                Text(
+                  "Not started",
+                  style: TextStyle(fontSize: 16),
+                ),
+              ],
+            ),
+          )),
+      Expanded(
+          flex: 33,
+          child: Container(
+            decoration: BoxDecoration(border: Border.all()),
+            alignment: Alignment.topCenter,
+            child: Column(
+              children: <Widget>[
+                FaIcon(FontAwesomeIcons.cog),
+                Text(
+                  "WIP",
```

Comment:
```suggestion
                  "Work In Progress",
```

Reply:
Der kan ikke stå Work In Progress på telefon skærme, men vi kan lave så der står WIP i portrait og Work In Progress i landscape